ui/lib/components/AutoRefreshButton/index.module.less

```diff
@@ -0,0 +1,12 @@
+.auto_refresh_btn {
+  :global {
+    .ant-dropdown-trigger {
+      width: auto;
+      padding: 0 10px;
+    }
+  }
+
+  .auto_refresh_secs {
+    font-size: 14px;
```

Comment:
Please use rem.

BTW I have no idea why this need to be specified explicitly. Maybe it should be something like button text size?